src/index.ts

```diff
@@ -19,7 +19,9 @@ import { YourSheet } from '/path-to-your-sheet/your-sheet.ts'
 const MySheet = new Sheet(
   'MySheet',
   {
-    exampleField: TextField(),
+    firstName: TextField(),
+    lastName: TextField(),
+    dob: DateField()
```

Comment:
Please make this "age", "NumberField".  Our DateField could use some refinement before putting it right in front of users.